<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XC飞行轨迹查看</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSliderStart">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>831</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderEnd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>821</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>终点</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>起点</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxEnd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>99</number>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>981</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabTrack">
     <attribute name="title">
      <string>飞行轨迹</string>
     </attribute>
     <widget class="QCheckBox" name="checkBoxP">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>升速</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>761</width>
        <height>571</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="QPushButton" name="pushButtonOpenFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开文件</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxGPS">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>航迹</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetFileInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>211</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBestUp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>最爽上升</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBestDown">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>最爽下降</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCurve">
     <attribute name="title">
      <string>统计曲线</string>
     </attribute>
     <widget class="QListWidget" name="listWidgetTrackInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>221</width>
        <height>441</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>229</red>
            <green>229</green>
            <blue>229</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>81</red>
            <green>132</green>
            <blue>44</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>229</red>
            <green>229</green>
            <blue>229</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>229</red>
            <green>229</green>
            <blue>229</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>81</red>
            <green>132</green>
            <blue>44</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>229</red>
            <green>229</green>
            <blue>229</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>229</red>
            <green>229</green>
            <blue>229</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxH">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>高度</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxDis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>距离</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxVV">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>上升速度</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxVH">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>水平速度</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxV3D">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Academy Engraved LET</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>三维速度</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>751</width>
        <height>571</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>horizontalSliderStart</tabstop>
  <tabstop>horizontalSliderEnd</tabstop>
  <tabstop>pushButtonOpenFile</tabstop>
  <tabstop>spinBoxStart</tabstop>
  <tabstop>spinBoxEnd</tabstop>
  <tabstop>checkBoxP</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>checkBoxGPS</tabstop>
  <tabstop>listWidgetFileInfo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
